configure.py

```diff
@@ -1051,6 +1065,8 @@ def is_arm_hard_float_abi():
 def host_arch_cc():
   """Host architecture check using the CC command."""
 
+  if sys.platform.startswith('zos'):
+    return 's390x';
```

Comment:
Accidental semicolon

```suggestion
    return 's390x'
```